### DISCUSSION TITLE
[ 问题咨询 ]关于如何通过percona-toolkit 提取AWS RDS的慢日志信息

### DISCUSSION THREAD

**Discussion by FSerKevin** [hhyo/Archery#1618] [Q&A]
配置信息是否有错误，该实例里面已经创建了archery库跟2张表  就是提取不了数据
![微信截图_20210724122510](https://user-images.githubusercontent.com/79827043/126857233-8d95d44a-4494-4522-b302-42f2d8b98a13.png)

执行sh文件一致提示这个报错
DBI connect(';host=/rdsdbdata/log/slowquery/mysql-slowquery.log;mysql_read_default_group=client','',...) failed: Unknown MySQL server host '/rdsdbdata/log/slowquery/mysql-slowquery.log' (2) at /usr/bin/pt-query-digest line 1001.
faile

**Comment by hhyo** [comment]
云数据库只能考虑通过api管理了
